pkg/loader/compose/v1v2.go

```diff
@@ -268,12 +269,123 @@ func libComposeToKomposeMapping(composeObject *project.Project) (kobject.Kompose
 			log.Infof("Service name in docker-compose has been changed from %q to %q", name, normalizeServiceNames(name))
 		}
 	}
+
+	// This will handle volume at earlier stage itself, it will resolves problems occurred due to `volumes_from` key
+	handleVolume(&komposeObject)
```

Comment:
@surajssd , loop iterate over a service one at a time. as we are dealing with `volumes_from` key here, we can't put this line into for loop